### ISSUE TITLE
Replica is not created when using create by resource.

### ISSUE THREAD

**Issue by sachu24** [CrunchyData/postgres-operator#1655]
**Describe the bug**
While creating postgres cluster with replica  using the create-by-resource example, the replica is not getting created.
On checking I could find the pgreplica crd is not getting created so I even tried manually adding a definition.

**To Reproduce**
1. Go to 'create-by-resource example'
2. Click on 'change the replica values from 0 to 1`
3. create the cluster

**Expected behavior**
Postgres cluster will be created with replica pods

**Please tell us about your environment:**

* Operating System: Centos
* Where is this running ( Local, Cloud Provider) : Local
* Storage being used (NFS, Hostpath, Gluster, etc): NFS
* Container Image Tag: centos7-11.7-4.2.2
* PostgreSQL Version: 11
* Platform (Docker, Kubernetes, OpenShift): Kubernetes

**Additional context**

**Comment by jatalocks** [comment]
Just opened an issue about the same problem. It seems like the "pgtask" for creation of a backup is not created, thus a replica cannot be created. I tried creating a backup using the API, but this seems to not work as well because pgBackRest has no "stanza" which is created by the job {cluster-name}-stanza-create. I tried created the job myself, but it also didn't work because it had permission issues accessing the pgBackRest pod using SSH, and I don't know how the API solves this issue.
